test/org/jetbrains/plugins/ideavim/ex/MapCommandTest.java

```diff
@@ -275,17 +275,29 @@ public void testMapLeader() {
   public void testAmbiguousMapping() {
     configureByText("\n");
     typeText(commandToKeys("nmap ,f iHello<Esc>"));
-    typeText(commandToKeys("nmap ,f2 iBye<Esc>"));
-    typeText(parseKeys(",fh"));
-    myFixture.checkResult("Hello\n");
+    typeText(commandToKeys("nmap ,fc iBye<Esc>"));
```

Comment:
I'm not really understand how this relates to the comment extension.

Reply:
I honestly don't remember the specifics, but based on the commit messages it looks like I ran into some issues with ambiguous mapping handling—possibly as part of my changes, or maybe just extra cases I wanted to make sure I didn't break.